.github/workflows/benchmarks.yml

```diff
@@ -92,6 +92,7 @@ jobs:
         with:
           run: |
             python -c "import pygmt; pygmt.show_versions()"
+            gmt set GMT_MAX_CORES=2
```

Comment:
remove this line?

Reply:
Ok, done at 31dce8e1740de6c1353324e8fbaffaae66e93e4e